rules/windows/process_creation/win_susp_pester.yml

```diff
@@ -0,0 +1,34 @@
+title: Execute Code with Pester.bat
+id: 59e938ff-0d6d-4dc3-b13f-36cc28734d4e
+description: Detects code execution via Pester.bat (Pester - Powershell Modulte for testing) 
+status: experimental
+references:
+    - https://twitter.com/Oddvarmoe/status/993383596244258816
+author: Julia Fomina, oscd.community
+date: 2020/10/08
+tags:
+    - attack.defense_evasion
+    - attack.t1216
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    powershell_module:
+        Image|endswith: '\powershell.exe'
+        CommandLine|contains|all:
+            - 'Pester'
+            - 'Get-Help'
+    cmd_execution:
+        Image|endswith: '\cmd.exe'
+        CommandLine|contains|all: 
+            - 'pester'
+            - ';'
+    get_help:
+        CommandLine|contains:
```

Comment:
The [LOLBIN page also says](https://github.com/LOLBAS-Project/LOLBAS/blob/94368c1e69a6ce5ce812f2b331c99b89a63791b9/yml/OSScripts/pester.yml#L7) that you can use just a question mark. And in the [old repository](https://github.com/api0cradle/LOLBAS/blob/master/OSScripts/pester.md), it is additionally said that there is still a help key without a slash

It looks like this:
![image](https://user-images.githubusercontent.com/61383585/95493846-b96b4180-09a5-11eb-8460-94a8dcc1997e.png)

Can you additionally check these parameters?

Reply:
Yeah, my fault
Just 'help' and '?' work as well
Fixed in the rule

Reply:
Cool! Thank you! 👍🏻